wagtail/admin/templates/wagtailadmin/pages/page_listing_header.html

```diff
@@ -44,6 +45,19 @@ <h1 class="w-sr-only">
                 {% comment %}Add an initial disabled & hidden submit button so that pressing 'enter' will not submit form. Reload not required as the content is dynamically loaded.{% endcomment %}
                 <div class="visuallyhidden"><input disabled type="submit" aria-hidden="true"/></div>
                 <div class="submit visuallyhidden"><input type="submit" value="Search" class="button" /></div>
+
+                {% dialog_toggle classname='button button-small w-mt-4' dialog_id='filters-dialog' text='Show filters' %}
+                {% dialog theme="floating" id="filters-dialog" title="Filters" dialog_root_selector="[data-search-form]" %}
+                    <div class="filterable__filters">
```

Comment:
I think we can remove this, it was mostly added to handle the grid layout when the filters are on the right side of the listing.

It also causes alignment issues for checkboxes:

## with the wrapper

<img width="221" alt="image" src="https://github.com/wagtail/wagtail/assets/6379424/188849ce-1ad3-46bb-abe5-16c9e4f2a3f6">


## without

<img width="221" alt="image" src="https://github.com/wagtail/wagtail/assets/6379424/738bac39-d5cf-4e83-8e53-fda2c38b5733">